common-develocity-gradle-configuration-groovy/settings.gradle

```diff
@@ -1,18 +1,16 @@
 plugins {
-    id 'com.gradle.enterprise' version '3.17'
+    id 'com.gradle.develocity' version '3.17'
     id 'com.gradle.common-custom-user-data-gradle-plugin' version '2'
 }
 
 def isCI = System.getenv('CI') != null // adjust to your CI provider
 
-gradleEnterprise {
+develocity {
     server = 'https://develocity-samples.gradle.com' // adjust to your Develocity server
     allowUntrustedServer = false // ensure a trusted certificate is configured
 
     buildScan {
-        capture { taskInputFiles = true }
```

Comment:
This is a good point. I think the reason behind having `allowUntrustedServer` here is that it may have been set to `true` during a trial period and therefore we want to ensure that it is set to `false` for production. This is a good discussion though, should we be more explicit about the other fields?

Reply:
I think sharing a streamlined version with only mandatory fields make sense

Reply:
So following this logic, we should also remove the `buildCache.local.isEnabled=true` logic. Do we agree on that?

Reply:
I would agree with that. I like the idea of this being the minimum viable recommended configuration. The less configuration there is the less chance for mistake.

Reply:
> So following this logic, we should also remove the buildCache.local.isEnabled=true logic. Do we agree on that?

I could go either way. On one hand, removing it does match the criteria we apply to the `develocity` configuration. On the other hand, I find it somewhat common that users don't understand that `org.gradle.caching=true` enables the local build cache, so I don't mind leaving it here to be explicit. But then we're not being consistent 🙂

Reply:
Ok here's my proposal: let's discuss the `buildCache.local.isEnabled=true` and `allowUntrustedServer = false` changes separately and merge this PR as is.